src/base/rss/rss_feed.cpp

```diff
@@ -79,6 +80,11 @@ Feed::Feed(const QUuid &uid, const QString &url, const QString &path, Session *s
 
     m_iconPath = storageDir / Path(uidHex + u".ico");
 
+    m_serializer = new Private::FeedSerializer;
+    m_serializer->moveToThread(m_session->workingThread());
+    connect(this, &Feed::destroyed, m_serializer, &Private::FeedSerializer::deleteLater);
```

Comment:
this may be causing rare crashes since the async invocation captures Feed, but it is destroyed before the serializer.

Reply:
It doesn't capture Feed.

Reply:
I meant in the async calls inside Feed::store and Feed::load.

Reply:
> > I meant in the async calls inside Feed::store and Feed::load.
> 
> Ah, I saw it. Need to think about it.

See #18031.